stanford_profile.info.yml

```diff
@@ -23,16 +23,13 @@ install:
   - drupal:field_ui
   - drupal:file
   - drupal:help
-  - drupal:history
-  - drupal:honeypot
   - drupal:image
   - drupal:menu_link_content
   - drupal:menu_ui
   - drupal:node
   - drupal:options
   - drupal:path
   - drupal:page_cache
-  - drupal:rdf
```

Comment:
Why not rdf?

Reply:
No need at this time

Reply:
theres a ticket to make more use of google assistant and alexa which will involve a whole lot of metadata work. we could wrap it into that one